queenbee/base/request.py

```diff
@@ -1,25 +1,40 @@
-import pathlib
 import os
+import pathlib
+import platform
 from urllib import request
-from typing import Union, Dict
-from .basemodel import BaseModel
+from typing import Dict
 
 USER_AGENT_STRING = 'Queenbee'
 
 
 def get_uri(url):
     """Resolve uri for urls and local files."""
-
-    if url.startswith('file:///'):
+    if url.startswith('file://'):
         return url.replace('\\', '/')
     elif url.startswith('http://') or url.startswith('https://'):
         return url
 
-    # a local file 
+    # a local file but is not formatted as local url
+    pre = 'file:///' if platform.system() == 'Windows' else 'file://'
```

Comment:
Since this logic is duplicated a couple times and cross-platform compatibility is so important here, what would you think about moving this to a small `env` module that handles setting the OS indicator once across the whole package?